src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -3,11 +3,13 @@
     "name": "Valid Person",
     "phone": "9482424",
     "email": "hans@example.com",
-    "address": "4th street"
+    "birthdayMonth": "1",
+    "reward": "1234"
```

Comment:
I like how you updated the sample data according to the new implementation 👍

Reply:
Thanks :^)